cpp/cmake_modules/BuildUtils.cmake

```diff
@@ -331,6 +332,14 @@ function(ADD_ARROW_TEST REL_TEST_NAME)
   endif()
   get_filename_component(TEST_NAME ${REL_TEST_NAME} NAME_WE)
 
+  if(ARG_PREFIX)
```

Comment:
What is the prefix for? Perhaps add a comment?

Reply:
OK